app.js

```diff
@@ -50,9 +52,6 @@ function initMap(){
 
     // New map
     var map = new google.maps.Map(document.getElementById('map'), options);
```

Comment:
Since you're using ES6 syntax, I would recommend only using `let` and `const` to be consistent.

Reply:
Good catch, thanks for the reminder! I definitely need to use consistent naming. I've updated all my variable naming to use `let` and `const`, following the recommendations you made in the comments below.